src/ModernHttpClient/iOS/NSUrlSessionHandler.cs

```diff
@@ -46,7 +46,7 @@ public class NativeMessageHandler : HttpClientHandler
         public NativeMessageHandler(): this(false, false) { }
         public NativeMessageHandler(bool throwOnCaptiveNetwork, bool customSSLVerification)
         {
-            var config = NSUrlSessionConfiguration.EphemeralSessionConfiguration;
+            var config = NSUrlSessionConfiguration.DefaultSessionConfiguration;
```

Comment:
There is documentation on the credential keys. So, I just set URL cache to null and merge.